README.md

```diff
@@ -78,8 +78,8 @@ Once that's finished, the application can be run as follows:
         --dns-godaddy-credentials /var/lib/letsencrypt/godaddy_credentials.ini \
         --keep-until-expiring --non-interactive --expand \
         --server https://acme-v02.api.letsencrypt.org/directory \
-        --agree-tos --email "webmaster@example.com" \
```

Comment:
Why remove the tos agreement and admin email?

Reply:
For two reasons:
1. In order to keep the non-docker and docker examples consistent
2. Keep the examples as short and simple as possible

Reply:
Without agreeing to the tos and setting email, it's going to ask for it in the command line. Either you have to leave the flags on the command line, or you have to o document going through the questions when certbot asks you for that information.

Reply:
I re-added this.